### DISCUSSION TITLE
Automating Energy Mix Modification for Country-Level LCA in Activity Browser

### DISCUSSION THREAD

**Discussion by LouiseBru** [LCA-ActivityBrowser/activity-browser#1290] [Q&A]
Hello everyone,

I am conducting a LCA of a process at the country level and need to modify the energy mix used. To avoid manually changing each activity, is there a way to automate this process? Can it be done using codes or a specific function within Activity Browser?

Thank you for your assistance.

**Comment by marc-vdm** [comment]
Hi @LouiseBru,

Assuming you're using ecoinvent, you can change the energy mix in the market process (e.g. 'market for electricity, high voltage, NL'. You should only have to change that one activity for all consumers of that activity to receive your new mix. 

That said, we generally recommend agianst editing processes in ecoinvent directly, as it may be hard to remember all your changes and you may need to use that database again and have results not in line with the original database.

What I would recommend instead is to use flow scenario calculations. You can find a [video tutorial here](https://www.youtube.com/watch?v=3LPcpV1G_jg). Please let me know if this resolves your issue!
